src/Pages/Applications/DevtronApps/service.ts

```diff
@@ -15,13 +15,37 @@
  *   limitations under the License.
  */
 
-import { ResourceKindType, get } from '@devtron-labs/devtron-fe-common-lib'
+import {
+    ResourceKindType,
+    ResponseType,
+    get,
+    getUrlWithSearchParams,
+    showError,
+} from '@devtron-labs/devtron-fe-common-lib'
 
-import { Routes } from '../../../config'
-import { AppConfigStatusResponse } from './service.types'
-import { DEFAULT_LANDING_STAGE } from './Details/AppConfigurations/appConfig.type'
+import { Routes } from '@Config/constants'
 
-export const getAppConfigStatus = (appId: number, resourceKind?: ResourceKindType): Promise<AppConfigStatusResponse> =>
+import { AppConfigStatusItemType } from './service.types'
+import { DEFAULT_LANDING_STAGE } from './Details/AppConfigurations/AppConfig.types'
+import { transformEnvConfig } from './Details/AppConfigurations/AppConfig.utils'
+
+export const getAppConfigStatus = (
+    appId: number,
+    resourceKind?: ResourceKindType,
+): Promise<ResponseType<AppConfigStatusItemType[]>> =>
     get(
-        `${Routes.APP_CONFIG_STATUS}?app-id=${appId}${resourceKind === ResourceKindType.job ? `&appType=${DEFAULT_LANDING_STAGE.JOB_VIEW}` : ''}`,
+        getUrlWithSearchParams(Routes.APP_CONFIG_STATUS, {
+            'app-id': appId,
```

Comment:
Why are we not using the camelCase here?

Reply:
required by api search param